src/app/components/common/dialog/jobs-manager/jobs-manager.store.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from '@angular/core';
+import { ComponentStore } from '@ngrx/component-store';
+import { EMPTY, Observable } from 'rxjs';
+import {
+  catchError, takeUntil, tap,
+} from 'rxjs/operators';
+import { JobsManagerState } from 'app/components/common/dialog/jobs-manager/interfaces/jobs-manager-state.interface';
+import { JobState } from 'app/enums/job-state.enum';
+import { Job } from 'app/interfaces/job.interface';
+import { EntityUtils } from 'app/pages/common/entity/utils';
+import { DialogService, WebSocketService } from 'app/services';
+
+const initialState: JobsManagerState = {
+  isLoading: false,
+  jobs: null,
+  runningJobs: null,
+  failedJobs: null,
+};
+
+@Injectable()
+export class JobsManagerStore extends ComponentStore<JobsManagerState> {
+  readonly limit = 5;
```

Comment:
Unsure about limit. I'd imagine I may want to see all running jobs and recent failed jobs.